ddl/db_integration_test.go

```diff
@@ -1169,7 +1169,7 @@ func (s *testIntegrationSuite5) TestBackwardCompatibility(c *C) {
 		TableID:    tbl.Meta().ID,
 		Type:       model.ActionAddIndex,
 		BinlogInfo: &model.HistoryInfo{},
-		Args:       []interface{}{unique, indexName, indexPartSpecifications, indexOption},
+		Args:       []interface{}{unique, false, indexName, indexPartSpecifications, indexOption},
```

Comment:
This may cause a problem when rolling upgrades. A new TiDB receives old args sent by old TiDB.

Reply:
Is there is a better way to pass one more arg to worker? I have no idea.

Reply:
Put it in the last arg

Reply:
Add new field in model.IndexInfo is backward compatible, i.e. a newer TiDB can handle older TiDB.
When the new version find the `Global` field is missing (by old TiDB), it set the value to false.

A old TiDB receive newer TiDB may be something wrong, because `Global` may be true and the old server do not know how to handle it, so we **must gurantee** there is no DDL during rolling update.